vcsi/vcsi.py

```diff
@@ -948,6 +948,7 @@ def mxn_type(string):
     try:
         split = string.split("x")
         m = int(split[0])
+        assert (m > 0)
```

Comment:
I like option 1. here. Catching different exception types and displaying appropriate error message in each case.